src/protocols/secure_channel/CASESession.cpp

```diff
@@ -1170,13 +1163,13 @@ CHIP_ERROR CASESession::ConstructSaltSigmaR2(const ByteSpan & rand, const P256Pu
     return CHIP_NO_ERROR;
 }
 
-CHIP_ERROR CASESession::ConstructSaltSigmaR3(const uint8_t * ipk, size_t ipkLen, MutableByteSpan & salt)
+CHIP_ERROR CASESession::ConstructSaltSigmaR3(const ByteSpan & ipk, MutableByteSpan & salt)
```

Comment:
Thanks for updating a lot of these to use ByteSpan

Reply:
No problem, needed to be done anyway.